airbyte-integrations/connectors/source-sentry/metadata.yaml

```diff
@@ -10,7 +10,16 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: cdaf146a-9b75-49fd-9dd2-9d64a0bb4781
-  dockerImageTag: 0.4.1
+  dockerImageTag: 1.0.0
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: "Source Sentry has been migrated to YAML Declarative Source."
```

Comment:
why is this a breaking change?

Reply:
+1. Seems like schemas have not changed?

Reply:
@girarda @natikgadzhi yes, actually this is not breaking change and low-code version of the connector is compatible with the previous one. According to the migration to low-code guide I thought we migrate all connectors to low-code as breaking change. If it's not I can remove breaking change docs from the pr.

Reply:
If the config, the state, and the schema are compatible, we shouldn't mark this as a breaking change. CC @katmarkham